### ISSUE TITLE
关于DCRNN模型中使用的数据集

### ISSUE THREAD

**Issue by Yiman-dot** [zouguojian/Traffic-speed-prediction#13]
您好，我看到您在DCRNN模型中似乎使用了YINCHUAN数据集进行训练，但在下载的文件中并没有找到这个数据集，请问该数据集可以在哪里获取

**Comment by zouguojian** [comment]
> 您好，我看到您在DCRNN模型中似乎使用了YINCHUAN数据集进行训练，但在下载的文件中并没有找到这个数据集，请问该数据集可以在哪里获取

Yep, you are right; we can not directly forward this private data to the public. However, if you are interested in this dataset, you can cooperate with our team and use this dataset. If you have any other problems, feel free to contact us.

Best wish, 
Guojian
